mmdet/core/evaluation/eval_hooks.py

```diff
@@ -114,15 +114,15 @@ def _do_evaluate(self, runner):
             tmpdir = osp.join(runner.work_dir, '.eval_hook')
 
         from mmdet.apis import multi_gpu_test
-        results = multi_gpu_test(
+        self.results = multi_gpu_test(
```

Comment:
Is this modification necessary?

Reply:
Hey @ZwwWayne this modification is necessary so that MMDetWandbHook can reuse the result.